selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/constant/TestTimeoutsConstants.java

```diff
@@ -24,4 +24,5 @@
   public static final int REDRAW_UI_ELEMENTS_TIMEOUT_SEC = 5 * MULTIPLE;
   public static final int ATTACHING_ELEM_TO_DOM_SEC = 3 * MULTIPLE;
   public static final int MINIMUM_SEC = MULTIPLE;
+  public static final int TIMEOUT_30_SEC = 30;
```

Comment:
Use 
https://github.com/eclipse/che/blob/3c9df765384a4703f7df0cba2003ed2c25d656dd/selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/constant/TestTimeoutsConstants.java#L21

Don't come up with new constant.

Reply:
@tolusha 30 or 40 sec doesn't matter ?

Reply:
It seems a matter of naming of constant. 30_SEC in name doesn't allow us to increase the value on demand without changing the name of variable which will lead to change code in many places. also, usage of MULTIPLE constant is can be useful in case of testing special product like an OCP or something else.